homeassistant/components/reddit/sensor.py

```diff
@@ -1,11 +1,20 @@
 """Support for Reddit."""
+from __future__ import annotations
+
 from datetime import timedelta
 import logging
+from typing import Any, Final
 
 import praw
+from praw.exceptions import PRAWException
+from praw.models import Subreddit
 import voluptuous as vol
+from voluptuous.schema_builder import Schema
 
-from homeassistant.components.sensor import PLATFORM_SCHEMA, SensorEntity
+from homeassistant.components.sensor import (
+    PLATFORM_SCHEMA as PARENT_PLATFORM_SCHEMA,
```

Comment:
This is breaking our general standard. Please remove the aliasing of PLATFORM_SCHEMA in this case.